examples/http_server_proxy.rs

```diff
@@ -53,10 +53,12 @@ async fn proxy(server_req: Request<Body>, target_url: Uri) -> Result<Response<Bo
             .expect("no errors could be in ResponseBuilder")
             .append(key, value.clone());
     }
-    // FIXME: Convert UnsyncBoxBody<Bytes, Error> to Response<Body>
-    server_resp
+    let resp_body = server_resp
         .body(client_resp.into_body().into_boxed_body())
-        .map_err(Error::from)
+        .map_err(Error::from)?;
+    let (resp_parts, resp_body) = resp_body.into_parts();
```

Comment:
This transformation is actually https://docs.rs/http/latest/http/response/struct.Response.html#method.map `server_resp.map(Body::from_http_body)`